src/applications/facility-locator/tests/components/UrgentCareAlert.unit.spec.jsx

```diff
@@ -1,52 +0,0 @@
-import React from 'react';
```

Comment:
This alert is not used; Slack thread here: https://dsva.slack.com/archives/C0FQSS30V/p1723150747496079